src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -1,12 +1,46 @@
 package frc.robot.subsystems.hood;
 
 import edu.wpi.first.math.geometry.Translation2d;
-import edu.wpi.first.units.Dimensionless;
-import edu.wpi.first.units.Measure;
-import edu.wpi.first.units.Units;
+import edu.wpi.first.units.*;
+import frc.robot.Constants;
+import lib.webconstants.LoggedTunableNumber;
 
 public class HoodConstants {
     public static Translation2d MECHANISM_2D_POSE = new Translation2d(1, 1);
-    public static double HOOD_LENGTH = 0.4; // m
+    public static Measure<Distance> HOOD_LENGTH = Units.Meters.of(0.4);
     public static Measure<Dimensionless> POSITION_TOLERANCE = Units.Percent.of(5);
+    public static double MAX_VELOCITY = 1; // RPS
+    public static double MAX_ACCELERATION = 4; // RPS squared
```

Comment:
Measure

Reply:
It can't be a measure, there is no rotational acceleration unit.

Reply:
`Measure<Velocity<Velocity<Angle>>` doesn't work?

Reply:
Oh you need a unit. You can use the `Per` method tho

Reply:
4d4a202